lib/index.js

```diff
@@ -87,13 +87,13 @@ const setup = ({customTextFormattings, parseFigureProps, customCaption} = {}) =>
       return <Article items={[]} articleProps={articleProps} />;
     },
     shouldUpdate: ({props, id}, nextProps) => {
-      const current = currentElement[id];
-      const next = renderArticleJson({ items: nextProps.items });
-      nextElement[id] = next;
       const itemsHasUpdated = props.items !== nextProps.items;
-      const htmlHasUpdated = itemsHasUpdated && next.innerHTML !== current.innerHTML;
+      if (itemsHasUpdated) {
+        const next = renderArticleJson({ items: nextProps.items });
```

Comment:
@ryanscottaudio well we're doing this here so that we only do the rendering when the items has changed, no need to re-render when items are unchanged.